src/components/CommentList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Comment from './Comment'
+
+function CommentList(props) {
+    return (
+        <div>
```

Comment:
use 2 whitespaces instead of 4
```suggestion
      <div>
```